src/ui/button.h

```diff
@@ -34,13 +34,15 @@ namespace ui {
     WidgetType behaviorType() const;
     // Signals
     obs::signal<void()> Click;
+    obs::signal<void()> RightClick;
```

Comment:
I think I would rename this to "ContextClick" instead. I took a look at some gui frameworks (unity and android) and liked this name because it is more generic, or less implementation oriented.

Reply:
We talked about this in the interview with @ckaiser, he saw the RightClick on ButtonSet so it's based on the existing naming.

Anyway we could use ContextClick in a future when a tablet-like UI is introduced. Because there are other elements that use "right click" like

https://github.com/aseprite/aseprite/blob/6f65acb54f0d739b996d107989762db85c386ed0/src/ui/base.h#L31

But I guess several renames will be needed.

Reply:
Yeah the addition was mostly for internal and user consistency, if everything is already mouse-focused there's going to have to be either some kind of abstraction layer on top that maps whatever other input exists (touchscreen/pen/etc) to what is essentially going to be a mouse, or there's gonna have to be a pretty big refactor of input stuff, the former is probably the easier route to start with since there's some UI conventions that'd take some more planning like holding buttons to trigger context menus or swiping gestures, but it's still kinda nice to be able to do those with a mouse too if available.